docs/TELEMETRY_SPECIFICATION.md

```
@@ -0,0 +1,1167 @@
+# Management Node (zstack.properties)
+Telemetry.enabled=true
+Telemetry.metricsEnabled=true
+Telemetry.prometheusPort=9464
+Telemetry.metricsCollectionIntervalSeconds=15
+Telemetry.maxTrackedSignatures=500
+```
+
+### 5.3 Prometheus Endpoint
+
+When enabled, metrics are exposed at:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)